### PR TITLE
[GLUTEN-6887][VL] Daily Update Velox Version (2024_10_12)

### PR DESCRIPTION
**Pull request by GlutenPerfBot**
Upstream Velox's New Commits:

```txt
5bedca0f3 by Kevin Wilfong, Add tests constructing maps with duplicate custom TimestampWithTimezones (11228)
c434ed8c6 by Kevin Wilfong, Add support for custom comparison in Presto's contains UDF (11227)
903ae35d3 by Kevin Wilfong, Add support for custom comparison in Presto's array_position UDF (11226)
0fba52159 by Kevin Wilfong, Add tests for array_union and array_remove with TimestampWithTimezone (11222)
f4ca8a60c by duanmeng, Ignore tracing of auxiliary operator (11220)
74a0db903 by Deepak Majeti, Remove connector factory registrations (8871)
daeff5943 by Jialiang Tan, Do not pre-allocate memory when init vector stream (11211)
```

### CODE REVIEW

**Comment by zhouyuan** [comment]
failures are related with
https://github.com/facebookincubator/velox/pull/8871

File: cpp/velox/compute/VeloxBackend.cc [hunk 1]
```diff
@@ -113,6 +115,8 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   initJolFilesystem();
   initCache();
   initConnector();
+  velox::parquet::registerParquetReaderFactory();
+  velox::parquet::registerParquetWriterFactory();
```
Comment (zhouyuan) [inline cpp/velox/compute/VeloxBackend.cc:119]: @JkSelf PTAL

Comment (JkSelf) [reply cpp/velox/compute/VeloxBackend.cc:119]: @zhouyuan Yes. We need to add this registration here due to this refactoring https://github.com/facebookincubator/velox/pull/8871.

**Review by zhouyuan — APPROVED** [review]
